cli/server/getAvailable.js

```diff
@@ -66,8 +66,8 @@ const getAvailableNarratives = async (localNarrativesPath) => {
   try {
     narratives = await readdir(localNarrativesPath);
     narratives = narratives
-      .filter((file) => file.endsWith(".md") && file!=="README.md")
-      .map((file) => file.replace(".md", ""))
+      .filter((file) => /\.(md|json)$/.test(file) && file !== "README.md")
```

Comment:
Narratives are markdown files only. See https://nextstrain.github.io/auspice/narratives/how-to-write fore more